.deepsource.toml

```diff
@@ -1 +1,30 @@
 version = 1
+
+exclude_patterns = [
+    "**/build/**",
+    "**/node_modules/**"
+]
```

Comment:
@oceanroleplay / @VictoriqueMoe i suppose this exclude pattern to should be able to handle exclusion of child packages related build/node_modules. Any other thing we should exclude?

Reply:
create another PR, if it does not work.